src/types/config.ts

```diff
@@ -136,3 +112,6 @@ export class UpdatableConfig implements Readonly<Config> {
     this.config = { ...this.config, ...newConfig }
   }
 }
+
```

Comment:
What the context of adding this?

Reply:
No context, shouldn't be there - removed